src/powershell/Public/ConvertTo-FinOpsSchema.ps1

```diff
@@ -0,0 +1,214 @@
+    # This will run a test to estimate the processing time and then prompt the user to continue. If the user enters Y or y, the operation will continue. If the user enters N or n, the operation will be aborted. 
+
+    .EXAMPLE
+    .\ConvertTo-FinOpsSchema.ps1 -ActualCost .\Detail_Account_196312_202309_en.csv -outputFilePath .\Detail_Account_196312_202309_en_filtered.csv -ExportAllColumns $false -RunTest $true -rows 1000 -columns 10 -Force
+    # This will run a test to estimate the processing time and then continue without prompting the user. It will export only the columns specified in the column mapping.
+#>
+
+<# Parameter Block #>
+param(
+    [Parameter(Mandatory = $true)]
+    [string]
+    $ActualCost,
+
+    [Parameter(Mandatory = $true)]
+    [string]
+    $outputFilePath,
+
+    [Parameter(Mandatory = $false)]
+    [bool]
+    $RunTest = $false,
+
+    [Parameter(Mandatory = $false)]
+    [ValidateRange(0, 100000)]
+    [int]
+    $rows = 1000,
+
+    [Parameter(Mandatory = $false)]
+    [ValidateRange(0, 20)]
+    [int]
+    $columns = 10,
+
+    [Parameter(Mandatory = $false)]
+    [bool]
+    $ExportAllColumns = $true,
+
+    [Parameter(Mandatory = $false)]
+    [switch]
+    $Force
+)
+
+# Define the input and output file paths
+$ActualCost=".\Detail_Account_196312_202309_en.csv"
+$outputFilePath=".\Detail_Account_196312_202309_en_filtered.csv"
+
+# Define the column mapping
+# The keys are the original column names, and the values are the new column names. If the value is the same as the key, the column will not be renamed.
+# If the value is not the same as the key, the column will be renamed and the value will be used as the new column name.
+# If the key is not present in the input data, the column will not be included in the output data.
+$columnMapping = @{
+    billingAccountId = 'BillingAccountId'
+    billingAccountName = 'BillingAccountName'
+    ftk_AccountType = 'ftk_AccountType'
+    BillingProfileId = 'BillingProfileId'
+    SubscriptionId = 'SubscriptionId'
+}
+
+<# Helper Functions #>
+function Add-AccountType {
+    [CmdletBinding()]
+    param 
+    (
+        [PSCustomObject]
+        $Object
+    )
+
+    $ftk_AccountType = switch ($true) {
+        { $Object.BillingAccountId -eq $Object.BillingProfileId } { "EA" }
+        { $null -eq $Object.BillingProfileId -or 
+          $Object.BillingProfileId -eq $Object.SubscriptionId } { "Other" }
+        default { "MCA" }
+    }
+    # Add or Update the ftk_AccountType property to the object.
+    if ($Object.PSObject.Properties["ftk_AccountType"]) {
+        $Object.PSObject.Properties["ftk_AccountType"].Value = $ftk_AccountType
+    } else {
+        $Object | Add-Member -MemberType NoteProperty -Name "ftk_AccountType" -Value $ftk_AccountType -Force
+    }
+    
+    return $Object
```

Comment:
All this can go away if we make it a getter. (Or we can just return the switch/if block directly above.
```suggestion
    return $ftk_AccountType
```

Reply:
tested it and it works. Will be included in my next commit.